bin/grant-server/main.go

```diff
@@ -234,6 +236,7 @@ func main() {
 		// iterate over jobs
 		for i := 0; i < job.Workers; i++ {
 			// spin up a job worker for each worker
+			logger.Info().Msg("starting job worker")
```

Comment:
Will be addressed here: https://github.com/brave-intl/bat-go/issues/360

I don't see this as a blocker, as this will only show for the finite number of job workers on startup.